ui/app/AppLayouts/Chat/panels/ContactListPanel.qml

```diff
@@ -31,14 +32,15 @@ ScrollView {
             image.isIdenticon: !!model.identicon
             visible: {
                 if (selectMode) {
-                    return !searchString || model.name.toLowerCase().includes(searchString)
+                    return (!searchString || model.name.toLowerCase().includes(searchString))
+                            && !chatContentModule.usersModule.model.isContactWithIdAdded(model.pubKey)
```

Comment:
@borismelnik this check here `!chatContentModule.usersModule.model.isContactWithIdAdded(model.pubKey)` causes other type of issue cause it will alway use list of users added to the active chat. Please fix this.